templates/iam_policy/draft_metadata_persistence_lambda.json.tpl

```diff
@@ -0,0 +1,45 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": [
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-draft-metadata-persistence-${environment}",
+        "arn:aws:logs:eu-west-2:${account_id}:log-group:/aws/lambda/tdr-draft-metadata-persistence-${environment}:log-stream:*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ssm:GetParameter"
+      ],
+      "Resource": "arn:aws:ssm:eu-west-2:${account_id}:parameter${parameter_name}"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetObject",
+        "s3:GetObjectTagging",
+        "s3:ListBucket",
+        "s3:PutObject",
+        "s3:PutObjectTagging"
```

Comment:
Maybe we can remove these action items as we won't be uploading any files to S3 buckets in `tdr-draft-metadata-persistence` lambda:
```
"s3:PutObject",
"s3:PutObjectTagging"
```

Reply:
removed